DESCRIPTION

```diff
@@ -34,6 +34,7 @@ Depends:
     R (>= 3.0)
 Imports:
     XML,
+    xml2,
```

Comment:
I think you should increment the version of the package for this change

Reply:
Will do as part of my final PR on this block of work. I'm thinking this change merits a 1.6.0 ?

Reply:
yes